src/pages/index.js

```diff
@@ -0,0 +1,278 @@
+import "./index.css";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation.js";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "9493559a-52de-43ba-8a3f-31c0f0008fae",
+    "Content-Type": "application/json",
+  },
+});
+
+const cardsList = document.querySelector(".cards__list");
+if (!cardsList) {
+  console.error("Error: .cards__list element not found!");
+}
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    console.log(cards);
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.append(cardElement);
+    });
+    document.querySelector(".profile__avatar").src = userInfo.avatar;
+    document.querySelector(".profile__name").textContent = userInfo.name;
+    document.querySelector(".profile__description").textContent =
```

Comment:
`document.querySelector(".profile__name")`
`document.querySelector(".profile__description")`


You've already found these 2 elements as `profileName` and `profileDescription`